web/libs/editor/src/tags/object/TimeSeries/README.md

```diff
@@ -132,7 +132,24 @@ Identical logic but uses scaled brush coordinate.
 * If click time is **inside** `brushRange` we call `setCursor(time)` – only cursor moves.
 * If outside – `_updateViewForTime` recentres view and may emit sync.
 
-### 4.5 Overview dragging behavior
+### 4.5 Click during playback synchronization
+* **Problem solved**: Previously, clicking during video playback would seek the video but TimeSeries cursor would continue from its old position.
+* **Solution**: When `isPlaying` is true and user clicks, the `restartPlaybackFromTime(time)` action:
+  * Cancels current animation frame
+  * Converts clicked time to relative seconds for playback state
+  * Updates `playStartPosition` and `playStartTime` to the clicked position
+  * Restarts the playback loop from the new position
+* **Result**: Both video and TimeSeries cursor jump to clicked position and continue playing synchronously.
+
+### 4.6 Shared click handling logic
+* `handleTimeSeriesMainAreaClick()` in `helpers.js` provides unified click handling for both:
```

Comment:
I don't think we need that as `handleMainAreaClick` worked in both places already.